src/mixins/document-mixin.ts

```diff
@@ -63,12 +63,16 @@ export default class DocumentMixin extends Vue {
     const pdfBufferData = await file.arrayBuffer()
     const pdfData = new Uint8Array(pdfBufferData) // put it in a Uint8Array
     const pdf = await pdfjsLib.getDocument({ data: pdfData })
-    const p1 = await pdf.getPage(1)
-    const [x, y, w, h] = p1._pageInfo.view
-    const width = w - x
-    const height = h - y
-    const isvalidPageSize = (width / pageSizeInfo.pointsPerInch === pageSizeInfo.width) &&
-      (height / pageSizeInfo.pointsPerInch === pageSizeInfo.height)
+    let isvalidPageSize = true
```

Comment:
Not sure how picky we are about case, but shouldn't it be "isValidPageSize"? (capital V)

Reply:
Yes, it should be a capital V, but I'll remove this line in the next commit.